Assets/MixedRealityToolkit/Utilities/Gltf/Serialization/ExportGltf.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.Toolkit.Utilities.Gltf.Serialization
+{
+    public class ExportGltf
+    {
+        // TODO Implement
```

Comment:
nit: it may make more sense to add this class later so folks don't accidentally see this class name and assume the functionality exists

Reply:
Would you rather me throw an unimplemented exception?

Reply:
I would suggest opening an issue/feature request after we get in the bulk gltf importer code. For now, I'd prefer if this class wasn't added so we don't generate any confusion around what we are intending to support. I would appreciate getting a gltf exporter created in the MRTK, but it may be a lower priority than other features/issues.

Reply:
Remaining todo items are listed in the description of the pull request.

Reply:
Are there any specific todo items that you think need to be completed before we check in this review? Do unchecked todos get translated into issues?

Reply:
Good question. 

Top list items: 
- MRTK Standard Shader Support
- Async Texture loading

The rest could be in a single task or broken up into tasks.